### ISSUE TITLE
Question : CorrDiff Loss Function (SSE or MSE)

### ISSUE THREAD

**Issue by maxfield-green** [NVIDIA/physicsnemo#864]
I noticed that the CorrDiff loss function in this repository differs from what was used in the [original CorrDiff paper](https://arxiv.org/abs/2309.15214), since the implementation here does not take the mean squared errors over pixels (MSE), but instead seems to back propagate the batch sum of squared errors over the full field, leading to larger unscaled losses.  I'm wondering if there was a reason for this or if there are plans to implement the loss as described in the paper in the future. Thanks!

**Comment by CharlelieLrt** [comment]
@maxfield-green the `sum` reduction (instead of `mean`), is inherited from the original EDM implementation (see [there](https://github.com/NVlabs/edm/blob/008a4e5316c8e3bfe61a62f874bddba254295afb/training/training_loop.py#L132) for details). This has been tested and does not make a difference.
